workers/analyzer/src/test/kotlin/AnalyzerEndpointTest.kt

```diff
@@ -357,7 +344,7 @@ class AnalyzerEndpointTest : KoinTest, StringSpec() {
                     resolvedJobConfigContext = null,
                     traceId = "trace-id",
                 )
-                coEvery { setupAuthentication(any()) } just runs
+                coEvery { setupAuthentication(any(), any()) } just runs
```

Comment:
Just to clarify, does this change mean that if any package manager is using the .netrc file this would not work anymore?
Also, IIRC the scanner uses the .netrc file to download source artifacts from private repositories, does this still work?

Reply:
Package managers relying on the .netrc file are indeed problematic. Are you aware of any that fall into this category? The ones we typically deal with expect their credentials in a dedicated configuration file.

The scanner uses the downloader to download source code artifacts and clone repositories. This involves the authenticator and will still work. There might be issues with other version control systems than Git which are exclusively invoked as command line tools. Working around this would require some more changes. For instance, I would like to change the ORT scanner to not create the downloader itself, but allow passing it as a parameter. Then ORT Server could use a wrapper that makes sure that credentials are properly set up before delegating to the actual download functionality.

Reply:
> Package managers relying on the .netrc file are indeed problematic. Are you aware of any that fall into this category? The ones we typically deal with expect their credentials in a dedicated configuration file.

I did not check, I vaguely remember Pip using the .netrc file but I think with the python-inspector that probably works differently anyway.